runtime/CSharp/src/Tree/Xpath/XPathLexer.cs

```diff
@@ -101,61 +98,34 @@ public override IVocabulary Vocabulary
 	}
 	private void ID_action(RuleContext _localctx, int actionIndex) {
 		switch (actionIndex) {
-		case 0:
+		case 0: 
```

Comment:
Why excess whitespaces appeared in the PR?

Reply:
generated code so I've no idea